spec/httpserver/handlers/ok_spec.clj

```diff
@@ -2,10 +2,10 @@
   (:require [speclj.core :refer :all]
             [httpserver.handlers.ok :as ok]))
 
-(import '(com.td.HttpServer HttpResponse))
-
-(describe "OkHandler"
-  (it "returns an HttpRequest with code 200"
-    (let [ok-handler (ok/new-ok-handler)
-      response (.generateResponse ok-handler)]
-      (should= 200 (.responseCode response)))))
+(describe "ok"
+  (it "the map returned has key :code value of 200"
+     (should= 200 (:code (ok/call "dummy"))))
+  (it "the map returned has key :body value of OK"
+     (should= "OK" (:body (ok/call "dummy"))))
+  (it "the map returned has a :header map with key Content-Type value of text/plain"
+      (should= (hash-map "Content-Type" "text/plain") (:headers (ok/call "dummy")))))
```

Comment:
Just a syntax/style note, you'll typically see `{}` instead of `hash-map`
```clojure
(= (hash-map "foo" "bar") {"foo" "bar"})
```

(a few places)